README.md

```diff
@@ -31,12 +31,13 @@ Phimp.me is a Photo App for Android that aims to replace proprietary photo appli
   
 ---
 
+Do you have many different apps in your Android Phone for the purpose of clicking, editing and sharing images? Hey guess what, now you don't need all of those because `Phimpme` is here, with all the features packaged into one. So go and clean your phone memory to save your space for other useful purposes.
```

Comment:
Name of the app is **Phimp.me**. Please correct this also if it is wrong.

Reply:
I agree with @mayank8318 . @jishnujayakumar Kindly do the required changes.